admin_guide/limits.adoc

```diff
@@ -280,33 +279,33 @@ Per image stream, the following must hold true if specified:
 |`*Max[openshift.io/image-tags]*`
 |`length( uniqueimagetags( imagestream.spec.tags ) )` less than or equal to `Max[openshift.io/image-tags]`
 
-Where `uniqueimagetags` returns unique references to images of given spec tags.
+`uniqueimagetags` returns unique references to images of given spec tags.
 
 |`*Max[openshift.io/images]*`
 |`length( uniqueimages( imagestream.status.tags ) )` less than or equal to `Max[openshift.io/images]`
 
-Where and `uniqueimages` returns unique image names found in status tags. The
-name equals to image's digest.
+`uniqueimages` returns unique image names found in status tags. The name equals
+image's digest.
 
 |===
 
-==== Counting of image references
+==== Counting of Image References
 
 Resource `openshift.io/image-tags` represents unique
 link:../dev_guide/managing_images.html#referencing-images-in-image-streams[image
 references]. Possible references are an `*ImageStreamTag*`, an
 `*ImageStreamImage*` and a `*DockerImage*`. They may be created using commands
 `oc tag` and `oc import-image` or by using
-link:../dev_guide/managing_images.html#adding-tag[tag
-tracking]. No distinction is done between internal and external references.
-Each unique reference tagged in the image stream's spec is counted just once
-though. It doesn't restrict pushes to an internal docker registry in any way
-but is useful for tag restriction.
+link:../dev_guide/managing_images.html#adding-tag[tag tracking]. No distinction
+is done between internal and external references. Each unique reference tagged
+in the image stream's specification is counted just once, however. It does not
```

Comment:
Can the "however" be moved to the start of the sentence? I think it messes with the word flow as it is.